app/controllers/api/console.php

```diff
@@ -71,7 +71,7 @@
     ->param('prompt', '', new Text(2000), 'Prompt. A string containing questions asked to the AI assistant.')
     ->inject('response')
     ->action(function (string $prompt, Response $response) {
-        $ch = curl_init('http://appwrite-assistant:3003/');
+        $ch = curl_init(System::getEnv("_APP_ASSISTANT_HOST"));
```

Comment:
we might wanna check if this env var is actually set, might be important for self-hosted instances.

There's also `_APP_ASSISTANT_ENABLED` that tells the console if the assistant is enabled, we might wanna check for this one too here 👍🏻 

We have a similar check here https://github.com/appwrite/appwrite/blob/main/app/controllers/api/account.php#L1789-L1791